cogs/tomatoes_cog/main.py

```
@@ -0,0 +1,56 @@
+        Throws tomatoes at a user.
+
+        Example usage:
+        `$tomato @Pablo`
+        `$tomato @Rai`
+        Or reply to a message with `$tomato`
+        """
+        user = None
+        
+        if len(ctx.message.mentions) > 0:
+            user = ctx.message.mentions[0]
+        elif ctx.message.reference != None:
```

Ruff (0.15.18)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)